django/contrib/auth/tests/urls_admin_cutom_pk.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
You have a typo in a file name `cutom` -> `custom`, and I'm not sure if this test need separate file